src/sentry/integrations/slack/views/unlink_team.py

```diff
@@ -70,13 +70,15 @@ def handle(self, request: Request, signed_params: str) -> Response:
             logger.error("slack.team.unlink.integrity-error", extra=e)
```

Comment:
I think this try/catch can't trigger now but we should still return a 404 if `external_teams` is empty.